project/MiMa.scala

```diff
@@ -939,7 +939,10 @@ object MiMa extends AutoPlugin {
         // #19872 double wildcard for actor deployment config
         ProblemFilters.exclude[IncompatibleMethTypeProblem]("akka.actor.Deployer.lookup"),
         ProblemFilters.exclude[DirectMissingMethodProblem]("akka.util.WildcardTree.apply"),
-        ProblemFilters.exclude[DirectMissingMethodProblem]("akka.util.WildcardTree.find")
+        ProblemFilters.exclude[DirectMissingMethodProblem]("akka.util.WildcardTree.find"),
+
+        // #21061 add circuit breaker exponential backoff support
+        ProblemFilters.exclude[IncompatibleMethTypeProblem]("akka.pattern.CircuitBreaker.this")
```

Comment:
we must make the change binary compatible, i.e. add the old constructor signature